content/source/docs/enterprise/vcs/bitbucket-server.html.md

```diff
@@ -30,6 +30,11 @@ TFE uses webhooks to get new configurations. To support this, Bitbucket Server n
 
 Leave the page open in a browser tab, and remain logged in as an admin user.
 
+### Important Notes: 
+
+- The Plugin is disabled by default. Clicking `Enabled` will toggle the plugin on. 
+- There is an option to configure a `webhook URL` on the plugin. Leave this optional field blank. Terraform Enterprise will dynamically update the `webhook URL` after the VCS connection is established.
```

Comment:
Would it make sense to move this up before "Leave the page open...", right after the screenshot?